backend/migrations/2023-02-24-030452_create_users/up.sql

```diff
@@ -1,6 +1,6 @@
 -- Your SQL goes here
 CREATE TABLE users (
     uid text primary key not null,
-    username text not null,
+    username text not null unique,
```

Comment:
why are you setting a max length in the model  but not for the actual database field?

Reply:
ah, good idea!